docs/Data_Submission_Portal/Users_Guide/Troubleshooting.md

```diff
@@ -0,0 +1,1236 @@
+                        "submitter_id":"Blood-00009-aliquot01_lane1_barcodeACGTAC_55.bam"
+                    }
+                ],
+            "valid":false,
+            "warnings":[]
+        }
+    ],
+    "entity_error_count":1,
+    "message":"Transaction aborted due to 1 invalid entity.",
+    "success":false,
+    "transaction_id":6372988,
+    "transactional_error_count":0,
+    "transactional_errors":[],
+    "updated_entity_count":0
+}
+```
+
+```Request1
+[
+    {
+        "data_category": "Sequencing Reads",
+        "data_format": "BAI",
+        "data_type": "Aligned Reads Index",
+        "gencode_version": "v36",
+        "project_id": "CPTAC-3",
+        "submitter_id": "Aligned_Reads_Index_000000",
+        "file_name": "1c5321a4-3b33-4787-9393-3c999940284f.rna_seq.genomic.gdc_realn.bam.bai",
+        "file_size": 5106896,
+        "md5sum": "250a27e69c3556311934d0d8daab2207",
+        "urls": "s3://cleversafe.service.consul/stage-submission-5/1c5321a4-3b33-4787-9393-3c999940284f/1c5321a4-3b33-4787-9393-3c999940284f.rna_seq.genomic.gdc_realn.bam.bai",
+        "aligned_reads_files": {
+            "submitter_id": "Aligned_Reads_000000"
+        },
+        "type": "aligned_reads_index"
+    }
+]
+```
+```Response1
+{
+  "cases_related_to_created_entities_count": 0,
+  "cases_related_to_updated_entities_count": 0,
+  "code": 400,
+  "created_entity_count": 0,
+  "entities":
+    [
+        {
+            "action":"create",
+            "errors":
+                [
+                    {
+                        "keys":["aligned_reads_files"],
+                        "message":"No link destination found for aligned_reads_files, unique_keys='[{'project_id': 'GDC-INTERNAL', 'submitter_id': 'Aligned_Reads_000000'}]'",
+                        "type":"INVALID_LINK"
+                    }
+                ],
+            "id":
+                [
+                    {
+                        "id":"001812ff-15e0-4c29-a3bb-582f81b46b2c"
+                    }
+                ],
+            "related_cases":[],
+            "type":"aligned_reads_index",
+            "unique_keys":
+                [
+                    {
+                        "project_id":"CPTAC-3",
+                        "submitter_id":"Aligned_Reads_Index_000000"
+                    }
+                ],
+            "valid":false,
+            "warnings":[]
+        }
+    ],
+    "entity_error_count":1,
+    "message":"Transaction aborted due to 1 invalid entity.",
+    "success":false,
+    "transaction_id":6372995,
+    "transactional_error_count":0,
+    "transactional_errors":[],
+    "updated_entity_count":0
+}
+```
+
+#### INVALID_PROPERTY Messages
+
+|Message|Explanation|Solution|
+| --- | --- | --- |
+| __Key '{property}' is not a valid property for type '{entity}'__ | The designated entity does not accept that property | Ensure the property is accepted for the entity by reviewing the GDC Data Dictionary |
```

Comment:
Link "GDC Data Dictionary" to https://docs.gdc.cancer.gov/Data_Dictionary/viewer/